src/components/SupportingPartners/index.js

```diff
@@ -0,0 +1,71 @@
+import { A } from "@commons-ui/core";
+import { Grid, Typography } from "@material-ui/core";
+import { makeStyles } from "@material-ui/core/styles";
+import Image from "next/image";
+import PropTypes from "prop-types";
+import React from "react";
+
+import Section from "@/pesayetu/components/Section";
+
+const useStyles = makeStyles(({ palette, typography }) => ({
+  root: {
+    paddingTop: typography.pxToRem(56.69),
+    paddingBottom: typography.pxToRem(80),
+    background: palette.grey.light,
+  },
+  section: {},
+  title: {
+    marginBottom: typography.pxToRem(49.38),
+  },
+  logoContainer: {},
+  link: {},
+  logo: {
+    margin: "0",
+  },
+}));
+function SupportingPartners({ title, items, ...props }) {
+  const classes = useStyles(props);
+
+  return (
+    <div className={classes.root}>
+      <Section className={classes.section}>
+        <Typography variant="h4" className={classes.title}>
```

Comment:
```suggestion
        <Typography variant="h3" className={classes.title}>
```

Reply:
No @KhadijaMahanga its supposed to be 20px which is `h4`
![image](https://user-images.githubusercontent.com/23483005/129674673-62810096-6bc4-4233-864f-32eec9476ffb.png)
![image](https://user-images.githubusercontent.com/23483005/129674890-54c51f0f-f375-45e9-b4d4-958cd85d47d4.png)